content/zh/docs/reference/scheduling/policies.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 调度策略
+content_type: concept
+weight: 10
+---
+<!--  
+title: Scheduling Policies
+content_type: concept
+weight: 10
+-->
+
+<!-- overview -->
+
+<!--  
+A scheduling Policy can be used to specify the *predicates* and *priorities*
+that the {{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}
+runs to [filter and score nodes](/docs/concepts/scheduling-eviction/kube-scheduler/#kube-scheduler-implementation),
+respectively.
+-->
+{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}
+根据调度策略指定的*断言（predicates）*和*优先级（priorities）*
+分别对节点进行[过滤和打分](/zh/docs/concepts/scheduling-eviction/kube-scheduler/#kube-scheduler-implementation)。
+
+<!--  
+You can set a scheduling policy by running
+`kube-scheduler --policy-config-file <filename>` or
+`kube-scheduler --policy-configmap <ConfigMap>`
+and using the [Policy type](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1?tab=doc#Policy).
+-->
+你可以通过执行 `kube-scheduler --policy-config-file <filename>` 或
+`kube-scheduler --policy-configmap <ConfigMap>` 
+设置并使用[调度策略](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1?tab=doc#Policy)。
+
+
+<!-- body -->
+
+<!-- ## Predicates -->
+## 断言    {#predicates}
+
+
+<!-- The following *predicates* implement filtering: -->
+以下*断言*实现了过滤接口：
+
+<!-- 
+- `PodFitsHostPorts`: Checks if a Node has free ports (the network protocol kind)
+  for the Pod ports the Pod is requesting. 
+-->
+- `PodFitsHostPorts`：检查 Pod 请求的端口（网络协议类型）在节点上是否可用。
+
+<!-- - `PodFitsHost`: Checks if a Pod specifies a specific Node by its hostname. -->
+- `PodFitsHost`：检查 Pod 是否通过主机名指定了 Node。
+
+<!-- 
+- `PodFitsResources`: Checks if the Node has free resources (eg, CPU and Memory)
+  to meet the requirement of the Pod.
+-->
+- `PodFitsResources`：检查节点的空闲资源（例如，CPU和内存）是否满足 Pod 的要求。
+
+<!--  
+- `MatchNodeSelector`: Checks if a Pod's Node {{< glossary_tooltip term_id="selector" >}}
+   matches the Node's {{< glossary_tooltip text="label(s)" term_id="label" >}}.
+-->
+- `MatchNodeSelector`：检查 Pod 的 Node {{< glossary_tooltip text="Selector" term_id="selector" >}}
```

Comment:
```suggestion
- `MatchNodeSelector`：检查 Pod 的节点{{< glossary_tooltip text="选择算符" term_id="selector" >}}
```

Reply:
memo: translate text in glossary_tooltip as well